tests/serializers/tenant/test_tenant_serializer.py

```diff
@@ -9,11 +9,11 @@ def test_serializer(self, create_tenant):
         tenant = create_tenant()
 
         assert TenantSerializer.serialize(tenant) == {
-                'id': tenant.id,
                 'firstName':tenant.firstName,
+                'id': tenant.id,
                 'lastName':tenant.lastName,
                 'phone': tenant.phone,
                 'unitNum': tenant.unitNum,
-                'created_at': Time.format_date(tenant.created_at),
-                'updated_at': Time.format_date(tenant.updated_at)
+                'created_at': Time.serialized_date_format(tenant.created_at),
+                'updated_at': None
```

Comment:
"None" is just for these tests because we are immediately creating the object and then serializing it, which means there won't be an updated_at value since there isn't a default for it.